e2e-tests/non-voting/conf/nonvoting-rs0.yml

```diff
@@ -0,0 +1,25 @@
+apiVersion: psmdb.percona.com/v1
+kind: PerconaServerMongoDB
+metadata:
+  name: nonvoting
+spec:
+  #platform: openshift
+  image:
+  mongod:
+    security:
+      enableEncryption: true
+  replsets:
+  - name: rs0
+    affinity:
+      antiAffinityTopologyKey: none
+    non_voting:
+      enabled: true
+      size: 3
```

Comment:
can we test all possible fields here? like labels, annotations, affinity etc

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/723/commits/66cd08a054fd300d8fb5568e76b469426de356f1